pages/strategy.md

```diff
@@ -0,0 +1,58 @@
+
+## Objective
+
+**Make Exercism scalable**
+
+At the time of writing, Exercism has 8,000 new submissions per week. Our mentors currently cannot keep up with the amount of submissions submitted. When they do keep up, the number of submissions almost immediately doubles, as students stick with a Language Track. We are growing by 300-400 new users per day. Many find Exercism too hard to use, but in the future, when that is fixed, we will expect to see the number of submissions double again. We have never actively marketed Exercism. All 300,000 members have come through word of mouth. When we start to market Exercism, we expect to see an increase in usage by two orders of magnitude. Our most important goal at this stage is being able to give our current students a timely mentoring experience, and be able to cope with further growth without overwhelming mentors.
+
+We aim to achieve this goal by the end of 2019.
+
+## Strategy
+
+**Acquire and retain more mentors by making mentoring rewarding and enjoyable**
+
+Currently, mentoring can be both fulfilling and frustrating. It can be very rewarding to help a student that wants to learn. On the flipside, it can be repetitive, students can be rude, there are few positive feedback loops, and it can feel too much like a job rather than a hobby. In order to retain our existing mentors, and to acquire new ones, we must make mentoring more enjoyable and more rewarding. We must make our mentors feel like that are part of something bigger them them, and that they are contributing to something valuable. A good parallel is wikipedia, where thousands of people contribute (from once a year, to dozens of times per day) because they feel like they are making a tangible difference to something that matters, and because the process itself is straight-forward but (strangely) rewarding.
+
+## Tactics
+
+### Track Anatomy Project
+
+Some solutions are fun to mentor, some are extremely challenging. For example, solutions that require maths lead to mentors trying to tutor someone on algebra, not on coding. Other solutions can be solved in such a variety of ways that the mentor ends up debating the right algorithm, not teaching the idioms of the language.
+
+The Track Anatomy Project will fix this by providing a framework to Track Maintainers, which they can work through in order to determine the optimal series of core exercises, and sensible related side exercises. In our prototype Track (Ruby), the result of the Track Anatomy Project saw a average reduction in waiting time from one week to half a day, and reduced the queue from hundreds to ~30 at any time. It also saw a doubling in the number of submissions per week and dramatically increased mentor retention, with the most active weekly number mentors of any track. Encoding this into a framework and applying it to all tracks should have a dramatic affect on mentor enjoyment.
```

Comment:
> by providing a framework to Track Maintainers

Does one "work through a framework"? Perhaps calling it a tool might better?

> The Track Anatomy Project **fixes** this by providing a **tool** to Track Maintainers, which they can **use** to determine...

Then again, calling it a framework at the end of the paragraph makes complete sense. 

> to all tracks should have a dramatic affect on mentor enjoyment.

Might be good to be explicit that this is positive:

> to all tracks should have a **dramatically positive** affect on mentor enjoyment.